python.wake

```diff
@@ -38,6 +38,7 @@ global def addPythonEnv pipDir plan =
   | editPlanVisible (installed ++ _)
   | addPlanRelativePath  "PYTHONPATH"  venv
   | addPlanRelativePath  "PATH" "{venv}/bin"
+  | setPlanEnvironmentVar "VIRTUAL_ENV"  venv
```

Comment:
Can you revert this change as well so that we only tackle all of it at once as opposed to having some of the changes within this PR?

Reply:
Shall do.